components/test-suite/src/loci/tests/testng/Configuration.java

```diff
@@ -108,6 +109,7 @@ public class Configuration {
   private static final String EXCITATION_WAVELENGTH_UNIT = "ExcitationWavelengthUnit_";
   private static final String DETECTOR = "Detector_";
   private static final String NAME = "Name";
+  private static final String UNFLATTENED_NAME = "Alternate_Name";
```

Comment:
Re-reading this PR, I wonder whether we should use a more explicit key for this configuration. `Unflattened_Name` is probably the closest to the current logic but it is a bit of a mouthful. Arguably we could also reverse the logic and store `Flattened_Name` for multi-resolution samples.

Reply:
Attribute name updated in 7ed65ecdbc. I'm not entirely sure whether reversing the logic is more or less confusing overall; doing that would have a bigger impact on configuration changes, as `Name` would change and `Flattened_Name` would be added (vs just adding `Unflattened_Name`).